src/m365/planner/commands/task/task-checklistitem-list.ts

```diff
@@ -0,0 +1,76 @@
+import { Logger } from "../../../../cli";
+import { CommandOption } from "../../../../Command";
+import { accessToken } from "../../../../utils";
+import GlobalOptions from "../../../../GlobalOptions";
+import Auth from "../../../../Auth";
+import request from "../../../../request";
+import GraphCommand from "../../../base/GraphCommand";
+import commands from "../../commands";
+import { PlannerChecklistItems } from "@microsoft/microsoft-graph-types";
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  taskId: string;
+}
+
+class PlannerTaskChecklistitemListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.TASK_CHECKLISTITEM_LIST;
+  }
+
+  public get description(): string {
+    return "Lists the checklist items of a Planner task.";
+  }
+
+  public commandAction(
+    logger: Logger,
+    args: CommandArgs,
+    cb: () => void
+  ): void {
+    if (
+      accessToken.isAppOnlyAccessToken(
+        Auth.service.accessTokens[this.resource].accessToken
+      )
+    ) {
+      this.handleError(
+        "This command does not support application permissions.",
+        logger,
+        cb
+      );
+      return;
+    }
+
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/planner/tasks/${encodeURIComponent(args.options.taskId)}/details?$select=checklist`,
+      headers: {
+        accept: "application/json;odata.metadata=none"
+      },
+      responseType: "json"
+    };
+
+    request.get(requestOptions).then(
+      (res: any): void => {
+        const checklistitem: PlannerChecklistItems = res.checklist;
+        logger.log(checklistitem);
```

Comment:
I wonder if it wouldn't be enough if we just do `logger.log(res.checklist);` here instead 🤔
@plamber maybe second opinion on that 🙏

Reply:
@Adam-it  - I was thinking to use the types but here it make sense. I have implemented the changes and is ready for review again.